### ISSUE TITLE
Docker related errors are not passed correctly and logs as <nil>

### ISSUE THREAD

**Issue by superseb** [rancher/rke#1918]
**RKE version:**
v1.0.4

**Steps to Reproduce:**
- Create a cluster and create an invalid configuration

```
nodes:
- address: x
  internal_address: x
  user: root
  role: [controlplane,worker,etcd]

services:
  kubeproxy:
    extra_binds:
    - "/lib/modules:/lib/modules"
```
**Results:**

```
FATA[0085] [workerPlane] Failed to bring up Worker Plane: [Failed to create [kube-proxy] container on host [x]: Failed to create Docker container [kube-proxy] on host [x]: <nil>] 
```

**Comment by deniseschannon** [comment]
Available in v1.1.0-rc9

**Comment by bmdepesa** [comment]
rke v1.1.0-rc11

* Created a cluster with the above configuration
* Error shows correctly:
```
FATA[0096] [workerPlane] Failed to bring up Worker Plane: [Failed to create [kube-proxy] container on host [x]: Failed to create Docker container [kube-proxy] on host [x]: Error response from daemon: Duplicate mount point '/lib/modules']
```
